docs/standard/security/generating-keys-for-encryption-and-decryption.md

```diff
@@ -10,73 +10,74 @@ helpviewer_keywords:
+ The following example shows the creation of a new instance of the default implementation class for the <xref:System.Security.Cryptography.Aes> algorithm.  
   
 ```vb  
-Dim tdes As TripleDESCryptoServiceProvider = new TripleDESCryptoServiceProvider()  
+Dim aes As Aes = Aes.Create()  
 ```  
   
 ```csharp  
-TripleDESCryptoServiceProvider tdes = new TripleDESCryptoServiceProvider();  
+Aes aes = Aes.Create();  
 ```  
   
  When the previous code is executed, a new key and IV are generated and placed in the **Key** and **IV** properties, respectively.  
   
  Sometimes you might need to generate multiple keys. In this situation, you can create a new instance of a class that implements a symmetric algorithm and then create a new key and IV by calling the **GenerateKey** and **GenerateIV** methods. The following code example illustrates how to create new keys and IVs after a new instance of the symmetric cryptographic class has been made.  
   
 ```vb  
-Dim tdes As TripleDESCryptoServiceProvider = new TripleDESCryptoServiceProvider()  
-tdes.GenerateIV()  
-tdes.GenerateKey()  
+Dim aes As Aes = Aes.Create()  
+aes.GenerateIV()  
+aes.GenerateKey()  
 ```  
   
 ```csharp  
-TripleDESCryptoServiceProvider tdes = new TripleDESCryptoServiceProvider();  
-tdes.GenerateIV();  
-tdes.GenerateKey();  
+Aes aes = Aes.Create();  
+aes.GenerateIV();  
+aes.GenerateKey();  
 ```  
   
- When the previous code is executed, a key and IV are generated when the new instance of **TripleDESCryptoServiceProvider** is made. Another key and IV are created when the **GenerateKey** and **GenerateIV** methods are called.  
+ When the preceding code is executed, a key and IV are generated when the new instance of **Aes** is made. Another key and IV are created when the **GenerateKey** and **GenerateIV** methods are called.
   
-## Asymmetric Keys  
- The .NET Framework provides the <xref:System.Security.Cryptography.RSACryptoServiceProvider> and <xref:System.Security.Cryptography.DSACryptoServiceProvider> classes for asymmetric encryption. These classes create a public/private key pair when you use the parameterless constructor to create a new instance. Asymmetric keys can be either stored for use in multiple sessions or generated for one session only. While the public key can be made generally available, the private key should be closely guarded.  
+## Asymmetric Keys
+
+ .NET provides the <xref:System.Security.Cryptography.RSA> and <xref:System.Security.Cryptography.DSA> classes for asymmetric encryption. These classes create a public/private key pair when you use the parameterless `Create()` method to create a new instance. Asymmetric keys can be either stored for use in multiple sessions or generated for one session only. While the public key can be made generally available, the private key should be closely guarded.  
```

Comment:
DSA isn't an encryption class, so the paragraph should be rewritten to remove DSA and change plural classes to singular class.